brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/ToscaTypePlanTransformer.java

```diff
@@ -167,4 +180,28 @@ private void assertAvailable() {
         }
     }
 
+    @Override
+    public double scoreForTypeDefinition(String formatCode, Object catalogData) {
+        return 0; // TODO: Not yet implemented
+    }
+
+    @Override
+    public List<RegisteredType> createFromTypeDefinition(String formatCode, Object catalogData) {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    protected double scoreForNullFormat(Object planData, RegisteredType type, RegisteredTypeLoadingContext context) {
+        return 0; // TODO: Not yet implemented
```

Comment:
How about an implementation of this? TOSCA's spec says

> In the TOSCA Simple Profile, TOSCA Service Templates must always have, as the first line of YAML, the keyword “tosca_definitions_version” with an associated TOSCA Namespace Alias value.

Following `CampTypePlanTransformer` again, we can call `RegisteredTypes.getAsYamlMap(planData)`and return `1` if the map contains `tosca_definitions_version` or `0` otherwise.